<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="menu_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="menu_screen">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="exit_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quest_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>퀘스트</string>
    </property>
   </widget>
   <widget class="QPushButton" name="map_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>지도</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>저장</string>
    </property>
   </widget>
   <widget class="QPushButton" name="out_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>나가기</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="chat">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>481</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>3</number>
   </property>
   <property name="text">
    <string>텍스트</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
